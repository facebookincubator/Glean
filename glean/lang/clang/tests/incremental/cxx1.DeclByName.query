query: cxx1.DeclByName _
transform:
  - sort
  - - [key,name_lowercase]
    - [key,kind]
    - [key,ident,macro,key]
