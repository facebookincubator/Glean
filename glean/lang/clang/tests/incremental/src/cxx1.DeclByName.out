[
  "@generated",
  {
    "key": {
      "name_lowercase": "a",
      "kind": 6,
      "ident": { "name": { "key": "a" } }
    }
  },
  {
    "key": {
      "name_lowercase": "b",
      "kind": 6,
      "ident": { "name": { "key": "b" } }
    }
  },
  {
    "key": {
      "name_lowercase": "bb",
      "kind": 6,
      "ident": { "name": { "key": "bb" } }
    }
  },
  {
    "key": {
      "name_lowercase": "c",
      "kind": 3,
      "ident": { "name": { "key": "C" } }
    }
  },
  {
    "key": {
      "name_lowercase": "c",
      "kind": 7,
      "ident": { "name": { "key": "c" } }
    }
  },
  {
    "key": {
      "name_lowercase": "f",
      "kind": 6,
      "ident": { "name": { "key": "f" } }
    }
  },
  {
    "key": {
      "name_lowercase": "test",
      "kind": 0,
      "ident": { "name": { "key": "test" } }
    }
  },
  {
    "key": {
      "name_lowercase": "x",
      "kind": 7,
      "ident": { "name": { "key": "x" } }
    }
  },
  {
    "key": {
      "name_lowercase": "y",
      "kind": 7,
      "ident": { "name": { "key": "y" } }
    }
  }
]